packages/@okta/vuepress-site/docs/guides/configure-direct-auth-grants/main/coobov/use-flow.md

```diff
@@ -71,12 +69,12 @@ curl --request POST \
   --url https://${yourOktaDomain}/oauth2/v1/token \
   --header 'accept: application/json' \
   --header 'content-type: application/x-www-form-urlencoded' \
-  --data 'client_id=${client_id}&scope=openid%20profile&grant_type=urn:okta:params:oauth:grant-type:oob&oob_code=&{oob_code}'
+  --data 'client_id=${client_id}&scope=openid%20profile&grant_type=urn:okta:params:oauth:grant-type:oob&oob_code=${oob_code}'
 ```
 
 Note the parameters that are passed:
 
-- `client_id`: Matches the client ID of the native application that you created in the [Set up your app](#set-up-your-app) section. You can find it at the top of your app's **General** tab.
+- `client_id`: Matches the client ID of the mobile application that you created in the [Set up your app](#set-up-your-app) section. You can find it at the top of your app's **General** tab.
```

Comment:
why did you make this update? In fact it could just be removed to "the application"